doc/source/conf.py

```diff
@@ -19,27 +19,39 @@
 # All configuration values have a default; values that are commented out
 # serve to show the default.
 
-import sys, os
+import sys
+import os
 
 import nibabel
 
+# Check for external Sphinx extensions we depend on
+try:
+    import texext
+except ImportError:
+    raise RuntimeError('Need to install "texext" package for doc build')
+try:
+    import numpydoc
+except ImportError:
+    raise RuntimeError('Need to install "numpydoc" package for doc build')
+
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 sys.path.append(os.path.abspath('../sphinxext'))
 
-# -- General configuration -----------------------------------------------------
+# -- General configuration ----------------------------------------------------
 
 # We load the nibabel release info into a dict by explicit execution
 rel = {}
-execfile('../../nibabel/info.py', rel)
+with open(os.path.join('..', '..', 'nibabel', 'info.py'), 'r') as fobj:
```

Comment:
I don't like this relative path, as it assumes a particular current working directory. I think using either `nibabel.__file__` or `__file__` to establish an absolute file position would be more robust.

Looks like there are other relative paths (adding to the path above), so I'd either ignore what I just said, or fix for all :)